src/components/ExacGeneSummary.vue

```diff
@@ -139,22 +141,21 @@ export default {
               p_li: this.round(hits.exac.all.p_li, 1),
               link: resp.request.responseURL
             };
+          } else {
+            this.$emit('show-exac', false);
           }
+
         })
         .catch((err) => {
           // eslint-disable-next-line
+          this.$emit('show-exac', false);
           console.log('mygene.info error', err.message);
```

Comment:
Probably need to keep the eslint-disable comment and the console.log lines together ;-)